packages/frontend/src/fixtures/stubs.ts

```diff
@@ -0,0 +1,127 @@
+export const trustRegistryStub = {
+  entities: [
+    {
+      name: 'Absa',
+      dids: ['did:indy:sovrin:staging:C279iyCR8wtKiPC8o9iPmb'],
+      logo_url:
+        'https://s3.eu-central-1.amazonaws.com/builds.eth.company/absa.svg',
+      domain: 'www.absa.africa',
+      role: ['issuer'],
+      credentials: [
+        'did:indy:sovrin:staging:C279iyCR8wtKiPC8o9iPmb/anoncreds/v0/SCHEMA/e-KYC/1.0.0',
+      ],
+      id: '8fa665b6-7fc5-4b0b-baee-6221b1844ec8',
+      createdAt: '2023-11-10T14:43:36.083Z',
+      updatedAt: '2023-11-20T16:49:58.307Z',
+    },
+    {
+      name: 'Absa 2',
+      dids: ['did:indy:sovrin:staging:C279iyCR8wtKiPC8o9iPmc'],
```

Comment:
I think we should use a different entity, with different did (should be unique per entity), logo domain etc.
Even the id is the same!